src/aics-image-viewer/components/App/index.tsx

```diff
@@ -1035,8 +1040,36 @@ export default class App extends React.Component<AppProps, AppState> {
   changeBoundingBoxShowing = (showing: boolean): void => this.changeUserSelection("showBoundingBox", showing);
 
   onResetCamera(): void {
-    if (this.state.view3d) {
-      this.state.view3d.resetCamera();
+    this.state.view3d?.resetCamera();
+  }
+
+  onClippingPanelOpen(open: boolean): void {
+    const CLIPPING_PANEL_HEIGHT = 130;
+    const CLIPPING_PANEL_OPEN_TIME = 300;
```

Comment:
is this related to some value associated with opening a antd Drawer?  I wonder if we could get the correct actual value from somewhere else.

Reply:
Drawer has a `afterVisibleChange` prop which might help here?